validator/src/journal/block_validator.rs

```diff
@@ -217,10 +220,14 @@ where
         transaction_index: T,
         view_factory: StateViewFactory,
     ) -> Self {
-        let (tx, rcv) = channel::<(Block, Sender<BlockValidationResult>)>();
+        let (tx1, rcv1) = channel::<(Block, Sender<BlockValidationResult>)>();
+        let (tx2, rcv2) = channel::<(Block, Sender<BlockValidationResult>)>();
         BlockValidator {
-            tx,
-            rx: Some(rcv),
+            tx1,
```

Comment:
Why not just a vec of tuples?

Reply:
Makes sense. Then there can be a configurable number of background threads.